java/client/src/main/java/glide/api/commands/SetBaseCommands.java

```diff
@@ -76,6 +76,24 @@ public interface SetBaseCommands {
      */
     CompletableFuture<Long> scard(String key);
 
+    /**
```

Comment:
do we want to add a message that for cluster mode keys must be in the same slot? and maybe add a test for it?

Reply:
This kind of comment would apply to all commands that take multiple keys, right?  Like SUNION, etc.  
If we do this, we need to split the documentation for standalone and cluster-mode, even though the signature is the same on both types.

Reply:
So not splitting, simply write 
In cluster mode, all keys should be mapped to the same hash slot. (Or something better)

This should be in all commands that have multiple keys, I'm pretty sure Bar's doc of how to review mentioned that

Reply:
You are correct.  @Yury-Fridlyand need to update docs. 

That being said, I'd prefer to split the javadoc into two.  It's cleaner that way, and then we don't have documentation for standalone that includes irrelevant content.  It's not a lot of extra work.

Reply:
That's not good, it's duplicating the code, this isn't the first multi key commands we implement, a short line of the documentation is enough, splitting it to 2 will cause confusion. 
I don't believe that there are users who don't know what cluster mode is even if they're using a standalone cluster

Reply:
This is just the front-end API.  There's very little beyond the documentation in these files.  
If the API is different for cluster-mode vs standalone, it makes sense for the API to be similar - it might seem like code duplication only because the method names are similar.

Reply:
I added a line to doc, it is visible for both standalone and cluster client. I also added a test for cross-slot case.
We created a task to process all these commands later by splitting them or adding documentation.